web/forms.py

```
@@ -832,38 +833,96 @@ def clean_topics(self):
-        max_length=100,
+    """Form for creating course draft by both authenticated and unauthenticated users."""
+
+    course_title = forms.CharField(
+        max_length=200,
+        label="Course Title",
         widget=TailwindInput(
             attrs={
-                "placeholder": "What would you like to teach?",
+                "placeholder": "Enter your course title",
                 "class": "block w-full border rounded p-2 focus:outline-none focus:ring-2 focus:ring-indigo-500",
             }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)